app/templates/gentelella/users/events/_table.html

```diff
@@ -72,15 +72,15 @@
                         {% if flag %}
                             {% for id, item in ticket_stats.iteritems() %}
                                 <div>
-                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span> 
+                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span>
                                 </div>
                                 <br>
                              {% endfor %}
                         {% else %}
                             {{ _("No Ticket Information") }}
                         {% endif %}
                     </td>
-                    <td><a href="{{ url }}" target="_blank">{{ url }}</a></td>
+                    <td style="width:200px;"><a href="{{ url }}" target="_blank">{{ url }}</a></td>
```

Comment:
@niranjan94 as I know little about js , checking jquery.dataTables.js not getting where to make the change.
will making change in th instead of td would work ? (checked on internet)
![screenshot from 2017-02-10 23 05 29](https://cloud.githubusercontent.com/assets/17119462/22837219/91a7e438-efe5-11e6-98da-e4aa90d9ece2.png)